models/yang/sonic/sonic-arp-ndp.yang

```diff
@@ -0,0 +1,155 @@
+module sonic-arp-ndp {
+    namespace "http://github.com/Azure/sonic-arp-ndp";
```

Comment:
Why not sonic-neighbor module instead of sonic-arp-ndp?

Reply:
This name is more descriptive.  sonic-neighbor can easily be misinterpreted as BGP neighbor.

Reply:
The name has been changed to 'sonic-neighbors'